src/modules/markets/components/markets-list.jsx

```diff
@@ -24,6 +24,10 @@ export default class MarketsList extends Component {
     outstandingReturns: PropTypes.bool,
     collectMarketCreatorFees: PropTypes.func,
     isMobile: PropTypes.bool,
+    isForking: PropTypes.bool,
+    isForkingMarketFinalized: PropTypes.bool,
+    forkingMarket: PropTypes.string,
+    updateModal: PropTypes.func.isRequired,
```

Comment:
same .isRequired before optionals.

Ask me questions if not clear

instead of pushing these properties down the hierarchy.  See comment above.

remove propTypes from this file